src/common/pf_lldp.h

```diff
@@ -430,6 +430,16 @@ int pf_lldp_recv (
    pnal_buf_t * p_frame_buf,
    uint16_t offset);
 
+/**
+ * Check if an alias name matches any of the LLDP peers.
+ *
+ * @param net              InOut: The p-net stack instance
+ * @param alias            In:    Alias to be compared with LLDP peers
```

Comment:
`* @param alias            In:    Alias to be compared with LLDP peers. Null terminated string.`

Reply:
done